packages/cli-lib/lib/logs.js

```diff
@@ -5,32 +5,36 @@ const { logger, Styles } = require('../logger');
 const SEPARATOR = ' - ';
 const LOG_STATUS_COLORS = {
   SUCCESS: Styles.success,
+  ERROR: Styles.error,
   UNHANDLED_ERROR: Styles.error,
   HANDLED_ERROR: Styles.error,
 };
 
-const formatError = log => {
-  return `${log.error.type}: ${log.error.message}\n${formatStackTrace(log)}`;
+const errorHandler = (log, options) => {
+  return `${formatLogHeader(log, options)}${
+    options.compact ? '' : `\n${formatError(log)}`
+  }`;
 };
 
 const logHandler = {
-  UNHANDLED_ERROR: (log, options) => {
-    return `${formatLogHeader(log, options)}${
-      options.compact ? '' : `\n${formatError(log)}`
-    }`;
-  },
-  HANDLED_ERROR: (log, options) => {
-    return `${formatLogHeader(log, options)}${
-      options.compact ? '' : `\n${formatError(log)}`
-    }`;
-  },
+  ERROR: errorHandler,
+  UNHANDLED_ERROR: errorHandler,
+  HANDLED_ERROR: errorHandler,
   SUCCESS: (log, options) => {
     return `${formatLogHeader(log, options)}${
       options.compact ? '' : `\n${log.log}`
     }`;
   },
 };
 
+const formatError = log => {
```

Comment:
Would it be good to put a try catch around the base error handler? Just to ensure we aren't breaking the customer

Reply:
There is one in the base that should catch https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli-lib/lib/logs.js#L62-L66